pom.xml

```diff
@@ -1212,21 +1213,8 @@ under the License.
         <reportSets>
           <reportSet>
             <reports>
+              <!-- only generate index by default, all other reports only active with profile "reporting" -->
               <report>index</report>
-              <report>summary</report>
-              <report>dependency-info</report>
-              <report>modules</report>
-              <report>team</report>
-              <report>scm</report>
-              <report>issue-management</report>
-              <report>mailing-lists</report>
-              <report>dependency-management</report>
-              <report>dependencies</report>
-              <report>dependency-convergence</report>
-              <report>ci-management</report>
-              <report>plugin-management</report>
-              <report>plugins</report>
-              <report>distribution-management</report>
```

Comment:
Should be for next PR - it is out of scope for me

Reply:
Created https://issues.apache.org/jira/browse/MPOM-358 and removed from this PR.